aws/_base/main.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 3.27"
+      version = "~> 3.72.0"
```

Comment:
I'm not sure what the meaning of `~>` is but doesn't look like `max version`. If that's the meaning then without this patch the maximum should have been `3.27`, no ?

Reply:
I'm not exactly sure either, looking at their documentation https://www.terraform.io/language/providers/requirements#version-constraints it should have but I guess it's because it was missing the third value `.0` it didn't take work? This works though, version `3.72.0` is being installed even though `3.73.0` is available